frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx

```diff
@@ -107,6 +108,24 @@ export const OidcAuth = () => {
         <>
             <Grid container sx={{ mb: 3 }}>
                 <Grid item md={12}>
+                    <ConditionallyRender
+                        condition={oidcLocked === true}
```

Comment:
No strong feelings either way, just an alternative.
```suggestion
                        condition={Boolean(oidcLocked)}
```